docs/rst/microros_documentation/benchmarking/benchmarking.rst

```diff
@@ -0,0 +1,169 @@
+.. _tutorials_micro_benchmarking:
+
+Benchmarking
+============
+
+.. contents::
+    :depth: 2
+    :local:
+    :backlinks: none
+
+
+This tutorial will give insight on micro-ROS benchmarking on different topics:
+
+- Memory profiling: Static, dynamic and stack.
+
+- Data throughput on different transports.
+
+- Latency between micro-ROS Client and Agent.
+
+This tests has been performed on a `Renesas EK-RA6M5 board <https://www.renesas.com/us/en/products/microcontrollers-microprocessors/ra-cortex-m-mcus/ek-ra6m5-evaluation-kit-ra6m5-mcu-group>`_ using :ref:`micro-ROS for Renesas e2 Studio <micro_ros_renesas_e2_studio>` build system.
+
+Memory Profiling
+----------------
+
+This section will cover micro-ROS memory usage on the most basic entity types. The test on this section have been performed using the provided UDP transport with FreeRTOS + TCP as network stack.
+
+Profiling methodology
+^^^^^^^^^^^^^^^^^^^^^
+
+The memory profile has been performed with the following configuration:
+
+- Reliable entities with a fixed topic size.
+- UDP transport (FreeRTOS + TCP).
```

Comment:
Link to FreeRTOS + TCP page ?

Reply:
https://github.com/eProsima/vulcanexus/pull/105/commits/db5c92b76a07451caa8aad2efad2fa99305a7004